ouroboros-consensus-cardano-tools/src/Cardano/Tools/DBAnalyser/Block/Cardano.hs

```diff
@@ -185,13 +192,14 @@ instance AdjustFilePaths CardanoConfig where
             byronGenesisPath    = f $ byronGenesisPath cc
           , shelleyGenesisPath  = f $ shelleyGenesisPath cc
           , alonzoGenesisPath   = f $ alonzoGenesisPath cc
+          , conwayGenesisPath   = f $ conwayGenesisPath cc
           }
 
 -- | Shelley transition arguments
```

Comment:
One thing that I find really confusing is why we don't have mention to Mary or Allegra here? Eg `MaryGenesis` :thinking:

Reply:
In this context, the "genesis data" is the data you need to start the block chain. Relatedly, genesis data would also be necessary to _identify_ "which Cardano chain" you were discussing, if there were more than one. And there could be: someone could start a new chain using the exact same code as ours, but starting with their own genesis block.

Byron, Shelley, Alonzo, and Conway are the only eras that have genesis data. The actual genesis block is a Byron block. In order to transition to Shelley, we needed to add some additional genesis data (eg some initial values of new protocol parametrers like `d`). Similarly in Alonzo (eg Plutus interpreter parameters/limits), and finally in Conway too (ie keys of the new genesis delegates).

In contrast, the Allegra, Mary, and Babbage eras did not introduce any new genesis data.

Does that explanation answer your question? @dnadales

Reply:
@nfrisby all we have at the moment is the [CIP-1694](https://github.com/cardano-foundation/CIPs/pull/380), which is still in design and which describes the next two ledger eras. There is a very high level description in the CIP about the [initial implementation](https://github.com/JaredCorduan/CIPs/blob/voltaire-v1/CIP-1694/README.md#initial-implementation-stage) plan, which is what will be included in conway.

Reply:
Thanks, Jared. I'm resolving this GitHub comment thread, because we know that Conway does have new Genesis data, so the simplification I was hoping for is not an option :(